src/Encamina.Enmarcha.SemanticKernel.Connectors.Document/Encamina.Enmarcha.SemanticKernel.Connectors.Document.csproj

```diff
@@ -20,6 +20,7 @@
     
   <ItemGroup>
     <PackageReference Include="ExcelNumberFormat" Version="1.1.0" />
+    <PackageReference Include="Microsoft.SemanticKernel.Abstractions" Version="1.15.0" />
     <PackageReference Include="Microsoft.SemanticKernel.Plugins.Document" Version="1.10.0-alpha" />
```

Comment:
I didn't notice it earlier, but I think the issue with the version `1.10.0` and the explicit version of `Microsoft.SemanticKernel.Abstractions` in line `23` is that the dependency in line `24` should be to versión `1.15.0` of SK instead of `1.10.0-alpha`.

Perhaps updating --> https://www.nuget.org/packages/Microsoft.SemanticKernel.Plugins.Document/1.15.0-alpha

Reply:
Yes, that was it :) I've changed it, thank you!